hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/safemode/SCMSafeModeManager.java

```diff
@@ -190,6 +196,18 @@ public synchronized void validateSafeModeExitRules(String ruleName,
    */
   @VisibleForTesting
   public void exitSafeMode(EventPublisher eventQueue) {
+    // Wait a while for as many as new pipelines to be ready
+    if (createPipelineInSafeMode) {
+      long sleepTime = config.getTimeDuration(
+          HddsConfigKeys.HDDS_SCM_WAIT_TIME_AFTER_SAFE_MODE_EXIT,
+          HddsConfigKeys.HDDS_SCM_WAIT_TIME_AFTER_SAFE_MODE_EXIT_DEFAULT,
+          TimeUnit.MILLISECONDS);
+      try {
+        Thread.sleep(sleepTime);
```

Comment:
This is for unit test. Some existing unit tests will fail because they want to write data once the miniOzone cluster exits the safemode and ready. But not all pipelines are ready to serve the write request after safemode.  
Maybe I can move this logic to miniOzoneCluster.